.buildscript/android-ui-tests.gradle

```diff
@@ -1,7 +1,6 @@
 android {
   defaultConfig {
     testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-    testInstrumentationRunnerArgument "listener", "leakcanary.FailTestOnLeakRunListener"
```

Comment:
I'm assuming removing this was somehow related to the upgrades?

Reply:
Ahhh it's because we're now chaining `DetectLeaksAfterTestSuccess` with the test rule.